keps/sig-storage/1710-selinux-relabeling/kep.yaml

```diff
@@ -26,6 +26,7 @@ milestone:
   stable: "v1.31" # Very optimistic plan for SELinuxMountReadWriteOncePod GA
 
   # alpha: "v1.30"  # SELinuxMount
+  # alpha: "v1.32"  # SELinuxChangePolicy
```

Comment:
Just a small question - should the latest milestone be updated to `latest-milestone: "v1.32"` to align with this update?

Below is the reference from the KEP [template](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml), so wanted to clarify. Apologies for any confusion on my part!

https://github.com/kubernetes/enhancements/blob/3e50abe0a97ea57f79e05557ac316421deceb800/keps/NNNN-kep-template/kep.yaml#L27-L30

Reply:
good point, fixed